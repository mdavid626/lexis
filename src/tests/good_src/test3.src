/' Program 3: Prace s retezci a vestavenymi funkcemi '/

Scope 'Hlavni telo programu
  Dim str1 As String
  Dim str2 As String
  Dim p As Integer
  
  str1 = "Toto je nejaky text"
  str2 = str1 + ", ktery jeste trochu obohatime"
  Print str1; "\n"; str2; "\n";
  p = find(str2, "text")
  Print "Pozice retezce \"text\" v retezci str2:"; p; "\n";
  Print "Zadejte nejakou posloupnost vsech malych pismen a-h, ";
  Print "pricemz se pismena nesmeji v posloupnosti opakovat";
  Input ; str1
  str2 = sort(str1)
  Do While (str2 <> "abcdefgh")
    Print "\nSpatne zadana posloupnost, zkuste znovu";
    Input ; str1
    str2 = sort(str1)
  Loop
  Print "\n";
End Scope
