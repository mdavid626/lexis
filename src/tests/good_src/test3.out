Toto je nejaky text
Toto je nejaky text, ktery jeste trochu obohatime
Pozice retezce "text" v retezci str2: 16
Zadejte nejakou posloupnost vsech malych pismen a-h, pricemz se pismena nesmeji v posloupnosti opakovat? 
